core/src/main/scala/cats/instances/sortedMap.scala

```diff
@@ -19,23 +17,22 @@ trait SortedMapInstances extends SortedMapInstances2 {
   def catsStdCommutativeMonoidForSortedMap[K: Order, V: CommutativeSemigroup] =
     cats.kernel.instances.sortedMap.catsKernelStdCommutativeMonoidForSortedMap[K, V]
 
-  implicit def catsStdShowForSortedMap[A: Order, B](implicit showA: Show[A], showB: Show[B]): Show[SortedMap[A, B]] =
-    new Show[SortedMap[A, B]] {
-      def show(m: SortedMap[A, B]): String =
-        m.iterator
-          .map { case (a, b) => showA.show(a) + " -> " + showB.show(b) }
-          .mkString("SortedMap(", ", ", ")")
-    }
+  implicit def catsStdShowForSortedMap[A, B](implicit showA: Show[A], showB: Show[B]): Show[SortedMap[A, B]] =
+    _.iterator
+      .map { case (a, b) => showA.show(a) + " -> " + showB.show(b) }
+      .mkString("SortedMap(", ", ", ")")
+
+  @deprecated("Use catsStdShowForSortedMap override without Order", "2.2.0-M2")
```

Comment:
there are still a few. They can be a follow up though, as long as it happens before M3